latte-test/some_control.latte

```diff
@@ -1,5 +1,10 @@
-{$someVariable}
-
-{$someVariable->callMe()}
-
-{$someVariable->missingMethod()}
+<p>
+    {$dateTime|date: 1, 2}
```

Comment:
@todo replace magic call with static call:

```diff
-$this->template->date(...)
+Latte\Runtime\Filters::date(...)
```

Get the default filter/function map somewhere